### DISCUSSION TITLE
Compiled binary size question

### DISCUSSION THREAD

**Discussion by injiniero** [official-stockfish/Stockfish#3820] [Q&A] [answered]
Hi!
I have installed stockfish 11 on ubuntu 20.04 from official repositories.
The size of stockfish binary is 271Kb

I've downloaded and compiled the source code of stockfish 14 and the binary size is 46Mb

Why this big difference?

Thank you in advance!

**Comment by vondele** [comment, marked as answer]
the new binary has a 40+Mb neural net embedded. This is needed to have NNUE working easily (by default)

**Comment by niklasf** [comment]
Stockfish 11 is the last release before NNUE was introduced. All versions after that come with a large set of neural network parameters, optionally embedded into the binary.

---

vondele 6 seconds faster 😄
